freqtrade/freqtradebot.py

```diff
@@ -1275,36 +1368,35 @@ def update_trade_state(self, trade: Trade, order_id: str, action_order: Dict[str
             logger.warning('Unable to fetch order %s: %s', order_id, exception)
             return False
 
+        logger.info(f"Updating order from exchange: {order}")
```

Comment:
we should talk about verbosity ...

this PR adds quite some "info level" messages.
that's OK where they are needed / provide useful information for the user - but for example this one is not in my opinion (it's showing a dict directly - which is kinda ugly, usually).

if we need the order from the exchange (what the exchange answered) - we can use `log_responses` in the exchange class ... 

_for some background_ : I'm fighting to reduce verbosity - to avoid having "log spams" (which we still do in some scenarios) - where the same event (like a buy) triggers 3-5 messages - which all "almost" say the same thing (we're buying). All are at a different stage - but don't "really" provide anything useful - unless something goes wrong (which will have a different message anyway).

Reply:
Totally agree. Those logs were added for better monitoring of what's going on while investigating a bug that only happened in live, so needed other testers to be able to copy some logs to me without enabling the full debug verbosity. It's resolved now and we don't need so many logs at the moment.